Sources/MapboxCoreNavigation/ActiveNavigationFeedbackType.swift

```diff
@@ -148,27 +144,19 @@ public enum RouteQualitySubtype: String, CaseIterable {
     case alternativeRouteNotExpected
     case routeIncludedMissingRoads
     case routeHadRoadsTooNarrowToPass
-    case other
 }
 
 /// Enum denoting the subtypes of the  `Illegal Route` top-level category
 public enum IllegalRouteSubtype: String, CaseIterable {
     case routedDownAOneWay
     case turnWasNotAllowed
     case carsNotAllowedOnStreet
-    case turnAtIntersectionUnprotected
-    case other
 }
 
 /// Enum denoting the subtypes of the  `Road Closure` top-level category
 public enum RoadClosureSubtype: String, CaseIterable {
     case streetPermanentlyBlockedOff
     case roadMissingFromMap
```

Comment:
Aaron Baddley confirmed we need to remote this subcategory.

Reply:
Thanks, a missing road and a closed road are different things anyway.